servicenow/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.6.3",
+  "name": "servicenow",
+  "short_description": "servicenow description.",
+  "guid": "40c3b6e7-82dc-4d0c-90f4-be5bc726889e",
```

Comment:
Was this copied from somewhere? If so, we need to make sure this guid is unique

Reply:
yeah it's a unique one.